include/linux/regmap.h

```diff
@@ -503,11 +503,22 @@ struct regmap_range_cfg {
  * struct regmap_sdw_mbq_cfg - Configuration for Multi-Byte Quantities
  *
  * @mbq_size: Callback returning the actual size of the given register.
+ * @deferrable: Callback returning true if the hardware can defer
+ *              transactions to the given register.
```

Comment:
@charleskeepax we should add a comment that only SDCA controls are allowed to be deferred, and for SDCA devices the DisCo spec provides a list of all SDCA controls and if each control can be deferred with a _DSD property. In other words this callback could be implemented either with a hard-coded switch table in the driver or by collecting information from platform firmware.